<ui version="4.0" >
 <class>pqGlobalStreamingViewOptions</class>
 <widget class="QWidget" name="pqGlobalStreamingViewOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="General" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Number of Streamed Passes</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="StreamedPasses" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Enable Stream Messages</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QCheckBox" name="EnableStreamMessages" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Enable Prioritization</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QCheckBox" name="UsePrioritization" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>View Dependent Ordering</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QCheckBox" name="UseViewOrdering" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Piece Cache Limit</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLineEdit" name="PieceCacheLimit" />
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Piece Render Cutoff</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="PieceRenderCutoff" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
